web/src/lib/components/faces-page/suggest-merge.svelte

```diff
@@ -0,0 +1,117 @@
+<script lang="ts">
+  import CircleIconButton from '../elements/buttons/circle-icon-button.svelte';
+  import { createEventDispatcher } from 'svelte';
+  import Close from 'svelte-material-icons/Close.svelte';
+  import ImageThumbnail from '../assets/thumbnail/image-thumbnail.svelte';
+  import type { PersonResponseDto } from '../../../api/open-api';
+  import { api } from '@api';
+  import Merge from 'svelte-material-icons/Merge.svelte';
+  import Button from '../elements/buttons/button.svelte';
+  import ArrowLeft from 'svelte-material-icons/ArrowLeft.svelte';
+  const dispatch = createEventDispatcher();
+
+  export let personMerge1: PersonResponseDto;
+  export let personMerge2: PersonResponseDto;
+  export let potentialMergePeople: PersonResponseDto[];
```

Comment:
What are the differences between `personMerge1`, `personMerge2`, and `potentialMergePeople`?

Reply:
Maybe a cleaner interface would just be all people. Then you could have merge people be reactive here instead.

`$: mergePeople = people.find(person => person.name === personMerge1.name && !person.isHidden && person.id !== personMerge1.id)`

Reply:
> What are the differences between `personMerge1`, `personMerge2`, and `potentialMergePeople`?

On the modal, personMerge1 is the person on the left and personMerge2 the person on the right; so PersonMerge1 is the person to merge into PersonMerge2. When you click on PersonMerge2 it shows the list of other people with the same name (potentialMergePeople)